src/components/view/Contact/ContactProfileCard/index.js

```diff
@@ -0,0 +1,63 @@
+
+import React, { Component } from 'react';
+
+import {Card, CardActions, CardHeader, CardText} from 'material-ui/Card';
+
+class ContactProfileCard extends React.Component {
+
+  render() {
+    const { contact } = this.props;
+
+    const infoKeyStyle = {
+      color: '#90A4AE',
```

Comment:
Can you pull colors from muiThemable? See pattern in the PMC-2018 app if you need a reference. It involves some small tweaks to the controller for this as well, but should be worthwhile and would be a great learning experience (that we'll all have to follow). 

I'd recommend having ContactInfo set up the muiThemable and then pass the muiTheme to its sub-components.